src/Main.js

```diff
@@ -25,73 +27,86 @@ export default class Main extends Component{
             let returnedResponse = response.data[0];
     
-            this.returnedLocation(returnedResponse);
+            this.setState({searchedCity:returnedResponse});
+            
             this.requestForecast();
+            this.requestMovies();
         } catch (e) {
-            let modifiedResponse = e.response.status;
-            console.log(modifiedResponse);
-            this.setState({error: true});
-            this.setState({errorStatus:modifiedResponse});
+            if(e){
+                console.log(e.response)
+                this.setState({error: false});
+            }
         }
     }
 
-    // this will send to our server to gather forecast 
+    // create request for weather data
     requestForecast = async() =>{
-        let cityName = this.manipulateDiplayName(this.state.searchedCity.display_name);
 
 
         try{
-            let url = `http://localhost:3001/weather?city_name=${cityName}&lat=${this.state.searchedCity.lat}&lon=${this.state.searchedCity.lon}`;
-    
+            //create url from the searched city
+            let url = `http://localhost:3001/weather?lat=${this.state.searchedCity.lat}&lon=${this.state.searchedCity.lon}`;
+        
+            //send the request to the server
             let response = await axios.get(url);
-    
-            console.log(response.data);
-    
+        
             let forecastData = response.data
-    
+        
             this.setState({forecast:forecastData})
-    
+        
             console.log(this.state.forecast);
+
         }
-        catch (e){
+        catch(e){
             if(e){
-                let modifiedResponse = e.response.status;
-                console.log(modifiedResponse);
-                this.setState({error: true});
-                this.setState({errorStatus:modifiedResponse});
+                console.error('something went wrong');
             }
         }
     }
 
-    // this will make the name work for the search query
-    manipulateDiplayName = (displayName) => {
-        let split = displayName.split(',');
-        let splice = split.splice(0,1);
-        let displayCityName = splice[0];
-        return displayCityName;
+    requestMovies = async() =>{
 
-    }
 
-    //sets the searchedCity state from the returned data
-    returnedLocation = (returnedCities) =>{
-        this.setState({searchedCity:returnedCities});
+        try{
+            //create url from the searched city
+            let url = `http://localhost:3001/movies?keyword=${this.state.cityValue}`;
+        
+            //send the request to the server
+            let response = await axios.get(url);
+          
+            let movieArray = response.data
+        
+            this.setState({movies:movieArray})
+        
+            console.log(this.state.movies);
+
+        }
+        catch(e){
+            if(e){
+                console.error('something went wrong');
+            }
+        }
     }
```

Comment:
Really great job on lab!

Feedback here: If you can before merge, go through file and clean up whitespace and indentation for easier readability 👍